### DISCUSSION TITLE
Danish translation

### DISCUSSION THREAD

**Discussion by jacoline** [vesta-webtrees-2-custom-modules/vesta_extended_relationships#131] [Q&A] [answered]
I can not add the danish po and mo files

**Comment by ric2016** [comment, marked as answer]
You'll have to be more specific. What do you want to accomplish? What have you tried so far?

- Do you want to add a new translation language to Vesta? [That's done via weblate.](https://hosted.weblate.org/projects/vesta-webtrees-custom-modules/vesta-extended-relationships/)
- Or do you want to add further translation locally on your system only? That should work via po/mo files in the 'resources/lang' folder.
- Or do you want to contribute Danish relationship term logic specifically? That's a bit more complicated, but additions would be very welcome.

**Comment by jacoline** [comment]
I made a danish po and compiled to mo. Not finish yet. But for now. Only I have it. Just wanted to share it

Hent BlueMail til Android<https://bluemail.me>
Den 31. aug. 2024, fra 20.20, "Richard Cissée" ***@***.******@***.***>> skrev:

You'll have to be more specific. What do you want to accomplish? What have you tried so far?

  *   Do you want to add a new translation language to Vesta? That's done via weblate.<https://hosted.weblate.org/projects/vesta-webtrees-custom-modules/vesta-extended-relationships/>
  *   Or do you want to add further translation locally on your system only? That should work via po/mo files in the 'resources/lang' folder.
  *   Or do you want to contribute Danish relationship term logic specifically? That's a bit more complicated, but additions would be very welcome.

**Comment by ric2016** [comment]
In that case: Adding the files to the 'resources/lang' does not work?

**Comment by jacoline** [comment]
It does work. Wanted to share with other who need danish. And tried to upload to you

Hent BlueMail til Android<https://bluemail.me>
Den 1. sep. 2024, fra 08.32, "Richard Cissée" ***@***.******@***.***>> skrev:

In that case: Adding the files to the 'resources/lang' does not work?

**Comment by ric2016** [comment]
> Wanted to share with other who need danish. And tried to upload to you

Creating a pull request did not work? It is hard for be to help you if you do not explain the issue in more detail. In general, weblate is used for Vesta translations (see above).
